blog/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+from blog.models import User, Post, Commentary
```

Comment:
fix imports according to PEP8 (https://peps.python.org/pep-0008/#imports)
You should put a blank line between each group of imports